pex/resolve/lockfile/model.py

```diff
@@ -123,39 +115,3 @@ def extract_requirement(req):
     locked_resolves = attr.ib()  # type: SortedTuple[LockedResolve]
     local_project_requirement_mapping = attr.ib(eq=False)  # type: Mapping[str, Requirement]
     source = attr.ib(default=None, eq=False)  # type: Optional[str]
-
-    def select(self, targets):
```

Comment:
This was the entry-point to the out-dated lock selection logic export (and update) were using. Now all removed.